### PR TITLE
Handle boolean options as bools rather than as strings

### PR DESCRIPTION
**Pull request by RobertKrawitz**
Boolean options are currently handled as strings and compared textually against "True" and "False".  They should be parsed and then treated as bools.

### CODE REVIEW

**Comment by RobertKrawitz** [comment]
This will require full e2e testing.

**Review by ebattat — COMMENTED** [review]
@RobertKrawitz, Did you test it ?

File: benchmark_runner/workloads/workloads_operations.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def __init__(self):
         self._workload = self._environment_variables_dict.get('workload', '')
         self._build_version = self._environment_variables_dict.get('build_version', '')
         self._workloads_odf_pvc = list(self._environment_variables_dict.get('workloads_odf_pvc', ''))
-        self._odf_pvc = self._environment_variables_dict.get('odf_pvc', '')
+        self._odf_pvc = self._environment_variables_dict.get('odf_pvc', True)
```
Comment (gqlo) [inline benchmark_runner/workloads/workloads_operations.py:32]: By default is True?

File: benchmark_runner/workloads/workloads_operations.py [hunk 2]
```diff
@@ -40,7 +40,7 @@ def __init__(self):
         self._endpoint_url = self._environment_variables_dict.get('endpoint_url', '')
         self._run_artifacts_path = self._environment_variables_dict.get('run_artifacts_path', '')
         self._save_artifacts_local = self._environment_variables_dict.get('save_artifacts_local', '')
-        self._enable_prometheus_snapshot = self._environment_variables_dict.get('enable_prometheus_snapshot', '')
+        self._enable_prometheus_snapshot = self._environment_variables_dict.get('enable_prometheus_snapshot', True)
```
Comment (gqlo) [inline benchmark_runner/workloads/workloads_operations.py:43]: Same here

File: benchmark_runner/workloads/workloads_operations.py (hunk 1 above)
Comment (RobertKrawitz) [reply benchmark_runner/workloads/workloads_operations.py:32]: @ebattat I believe this is the current default?

File: benchmark_runner/workloads/workloads_operations.py (hunk 2 above)
Comment (RobertKrawitz) [reply benchmark_runner/workloads/workloads_operations.py:43]: @ebattat is this what's intended?

File: benchmark_runner/workloads/workloads_operations.py (hunk 1 above)
Comment (ebattat) [reply benchmark_runner/workloads/workloads_operations.py:32]: Yes, True in the current default

File: benchmark_runner/workloads/workloads_operations.py (hunk 2 above)
Comment (ebattat) [reply benchmark_runner/workloads/workloads_operations.py:43]: False is the default

Comment (RobertKrawitz) [reply benchmark_runner/workloads/workloads_operations.py]: Fixed.

File: benchmark_runner/common/template_operations/template_operations.py [hunk 3]
```diff
@@ -32,7 +32,7 @@ def __get_workload_template_kind(self):
         Kata shares templates with pod
         """
         if self.__workload_kind == 'kata' or self.__workload_kind == 'pod':
-            if self.__environment_variables_dict.get('kata_cpuoffline_workaround', '') == 'True':
+            if self.__environment_variables_dict.get('kata_cpuoffline_workaround', False):
```
Comment (ebattat) [inline benchmark_runner/common/template_operations/template_operations.py:35]: Default should be True, Am I right ?

**Review by ebattat — APPROVED** [review]

**Pull request merged**